exercises/numbers/isEven/isEven.js

```diff
@@ -36,4 +36,12 @@ if (require.main === module) {
   }
 }
 
+if (num % 2 === 0) {
+  console.log(isEven(num) === true);
```

Comment:
Calling this inside the function creates an infinite, recursive loop. When the JavaScript interpreter (`node`) sees an expression that it recognizes as a function call, like

```js
isEven(0)
```

is _immediately_ jumps to the function and starts running it. So when `node` gets to this line and sees `isEven(num)`, it will call `isEven` _again_. It won't get to the `return` line until `isEven` returns...but `isEven` calls itself again and so on and so on.

It's like the movie Inception. Every time you call a function, you go into that function's world. A function can call itself, recursively.

Remove the `console.log` lines from this function and it will work. If the stuff about the function calling itself doesn't make sense, don't worry about it. Recursion is one of the trickiest topics for beginners. Avoid calling a function from inside itself unless you have a good reason to do so.